src/viz/helpers/ui/history.coffee

```diff
@@ -8,13 +8,6 @@ module.exports = (vars) ->
 
   if !vars.small and vars.history.states.length > 0
```

Comment:
I think the code would be more succinct if this if statement was modified as such:
```js
if !vars.small and vars.history.value and vars.history.states.length > 0
```

The rest of the behavior should be able to stay the same.